client/components/shared/ImportModule.js

```diff
@@ -8,7 +8,7 @@ export default class ImportModule extends React.Component {
 
         let $importFile = document.getElementById('importFile');
         if (!$importFile || !$importFile.files || !$importFile.files.length) {
-            alert('Need to select a file!');
+            alert('No file selected');
```

Comment:
If you're willing to, can you add a translation key here and translate this string too?